.github/workflows/contracts.yml

```
@@ -5,17 +5,34 @@ on:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          filters: |
+            contracts_changed:
+              - 'contracts/**'
+              - 'common/**'
 
 concurrency:
   group: contracts-ci-${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
-jobs:
   test_contracts:
```

actionlint (1.7.9)

unexpected key "test_contracts" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)